tomster-player/app/templates/components/album-list.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="list-group sidebar">
+<div class="list-group {{if show-songs '' 'sidebar'}}">
```

Comment:
This should use `attrs.show-songs` instead and actually not render the whole section at all if it is `false`:

```hbs
{{#if show-songs}}
  <div class="list-group">
    {{#each attrs.albums as |album|}}
    …
{{/if}}
```